### PR TITLE
Wizard Rod Form

### PR DESCRIPTION
**Pull request by uraniummeltdown**
https://github.com/tgstation/tgstation/pull/21419

Adds a new wizard spell for 2 points, Rod Form. Using it turns you into an immovable rod for a few seconds, travelling in the direction you were facing. Can be upgraded to increase range and decrease cooldown.
Cooldown is 60 seconds, max upgraded is 20 seconds
:cl:
add: The wizard has a new spell, Rod Form, which allows him to transform into an immovable rod.
/:cl:

### CODE REVIEW

**Comment by taukausanake** [comment]
omg that sounds amazing. Screw using knock or wand of doors, just pop this and smash through everything and everyone

**Comment by Twinmold93** [comment]
Sounds EXTREMELY overpowered and strong for a wizard spell. Ei'Nath (Disintergrate) is already an insta-gib/kill spell that targets a singular person. The immovable rod, if it hits the person, instantly gibs the person... Then through distance gibs everyone else in a line and destroys the walls/breaches to space... Even if it was added, which because of merely how overpowered it is I don't feel it would be at all fair or fun for anyone going against a wizard as it's less avoidable than Ei'Nath is, honestly, that 2 points is even remotely a 'decent' price for it. This would have to take up almost ALL of your points (if not all), because it can kill so many people instantly, damages the station very much, moves you forward through ANYTHING, and is much harder to predict to avoid.

**Comment by taukausanake** [comment]
I agree it is a _little_ overpowered seeming but I think it's more fun than that. The only things I can think of for this though is how long the cooldown is. I don't know what the numbers mean in there. Give it a cooldown like Ei'Nath and it shouldn't be too bad I'd think

**Comment by Crazylemon64** [comment]
@Twinmold93 It doesn't instantly gib, but it does cause lethal brute damage and a dismember. The rod cannot be controlled once set in motion, and can only be used in the 4 cardinal directions. This makes it most powerful when cornered in a narrow corridor, like in maintenance. In more wide-open situations, however, it is not a guaranteed kill, unlike Ei Nath, since Ei Nath is only consumed once a person is hit by it, while this spell has no guarantee on that - especially if your opponent is moving around erratically like most robust folk. Its ability to move through basically anything also affords it utility as an escape spell - though since it leaves a following route behind itself, it leaves some things to be desired in comparison to other escape spells. It has a cooldown of 25 seconds

**Comment by Crazylemon64** [comment]
Icon conflicts

**Comment by SamHPurp** [comment]
In agreement with @Twinmold93 this sounds far too good.

**Comment by jughu** [comment]
I see it now

Vulphakin the penetrator

**Comment by uraniummeltdown** [comment]
Since Paradise is slower paced and medical is more involved than TG, I'll make the cooldown like Ei Nath. Being struck by a rod doesn't gib you, it does cause a lot of brute damage though

**Comment by taukausanake** [comment]
I believe rods used to gib you? It the far past at least. If down to a certain health I'd like to see people pop but I guess that's for another time

File: code/datums/spells/rod_form.dm [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/obj/effect/proc_holder/spell/targeted/rod_form
+	name = "Rod Form"
+	desc = "Take on the form of an immovable rod, destroying all in your path."
+	clothes_req = 1
+	human_req = 0
+	charge_max = 600
```
Comment (Fox-McCloud) [inline code/datums/spells/rod_form.dm]: 250

File: code/datums/spells/rod_form.dm [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	cooldown_min = 200
```
Comment (Fox-McCloud) [inline code/datums/spells/rod_form.dm]: 100

**Comment by Fox-McCloud** [comment]
This isn't nearly as good as it's made out to be. Will it be excellent 1v1? Absolutely. 

Will it be outclass Ei Nath? Not really. It can put you in awkward positions you can't really control; it can put you in lot of awkward situations and the directions it'll travel in are completely random every time it bumps into something.

**Comment by taukausanake** [comment]
It can send you away while also dealing some damage, mobs or station, and if you got jaunt then get out of situations you don't like ending in. I see this as a good plus to wizards either way

**Comment by jughu** [comment]
Wouldnt be better then fireball really fireball pretty much instakill you too

**Pull request merged**